workflows/genomes_generation.nf

```diff
@@ -54,7 +54,7 @@ ref_genome_index   = file("${ref_genome.parent}/*.fa.*")
 include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoftwareversions/main'
 include { FASTQC as FASTQC_BEFORE     } from '../modules/nf-core/fastqc/main'
 include { FASTQC as FASTQC_AFTER      } from '../modules/nf-core/fastqc/main'
-include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
+// include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
```

Comment:
add comment here, please, something like that module takes a lot of resources and would be improved in the future :D

Reply:
https://embl.atlassian.net/browse/EMG-8390